analytics.py

```diff
@@ -22,4 +22,14 @@ def collect_event(self, _db, license_pool, event_type, time=None, **kwargs):
         if not time:
             time = datetime.datetime.utcnow()
         for provider in self.providers:
-            provider.collect_event(_db, license_pool, event_type, time, **kwargs)
\ No newline at end of file
+            provider.collect_event(_db, license_pool, event_type, time, **kwargs)
+
+def format_age_range(r):
```

Comment:
This is very similar to Work.target_age_string. Is there a way to reuse/improve that code?